dev-qt/designer/designer-5.15.0_beta3.ebuild

```diff
@@ -44,7 +44,7 @@ src_configure() {
 	# run in the root directory.
 	# Related bugs: 633776, 676948, and 716514.
 	mkdir -p "${QT5_BUILD_DIR}" || die
-	qt5_qmake "${QT_BUILD_DIR}"
+	qt5_qmake
```

Comment:
indeed, arguments to `qt5_qmake` are passed via the `myqmakeargs` bash array.

is the intention here to run `qt5_qmake` in `$S`?

Reply:
I believe so.  Things do "just work", and I had originally copied one of the ebuilds that had it like so: https://bugs.gentoo.org/633776#c6

Then it somehow snuck into one of the recent commits, where it was supposed to be 'QT5_BUILD_DIR', and then I noticed it doesn't matter what it is... which I actually seem to have first noticed back here: https://bugs.gentoo.org/676948#c1

Never looked into it after that, until now.

Reply:
so it's not just about *root* vs *subdir* as the comment above seems to imply, but also `S` vs `QT5_BUILD_DIR`, i.e. in-source vs out-of-source. Can you clarify the comment please? (and TBH, this is another huge hack, I wonder what's the point of doing out-of-source if we still need to do all these hacks that touch `S`...)